PROJECT

```diff
@@ -0,0 +1,3 @@
+version: "2"
+domain: etcd.improbable.io
```

Comment:
Should the domain be `etcd.improbable.io` or `improbable.io`? The domain is a suffix used to form the group name, so I think if we used `etcd` as the group name and the domain as *improbable.io*, we'd end out with a sensible looking apiVersion like `etcd.improbable.io/v1alpha1` etc

Reply:
Agree we probably want an API version with `etcd.improbable.io`. While we're on the topic, how is the versioning supposed to work?

Reply:
With the new commit I changed `--domain` to be `improbable.io` as defined here.

Reply:
+1 the group should be `etcd.improbable.io` to stay consistent with some other custom resources that we are working on internally.

on versioning, i'd suggest: 
`v1alphaN` during dev, where `N` starts at 1 and increases with each breaking change
`v1betaN` once we're ready to start deploying to our `beta` environment, and migrating over our existing clusters
`v1` once we're ready to go to production/write blogpost etc.

thoughts?